.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+<!-- MANY MANY THANKS FOR YOUR CONTRIBUTION. ML5 ❤️S YOU! -->
+
+## → Description 📝
+
+A clear and concise description of what the pull request is about. Let us know if you are:
+
+- fixing a bug 🐛
+- adding an update 🛠
+- submitting a new feature 🆕 
+
+## → Screenshots 🖼
+(Any relevant screenshots, sketches, or helpful concept diagrams - these are helpful for our release notes and for getting people excited about your super cool feature and/or fix!)
+
+
+## → Relevant Example or Paired Pull Request to [ml5-examples](https://github.com/ml5js/ml5-examples) 🦄
+
+In order to help the ml5 team understand your pull request, we ask that you please submit an example to [ml5-examples](https://github.com/ml5js/ml5-examples) showcasing how your integrated feature works. You're also welcome to show how this works by providing code that can be easily run by others. 
```

Comment:
Maybe mention the `development` branch here?

Reply:
ooh! yes. Maybe something like:

## → Use the right branch 🌲

- for pull requests related to 🆕**new features** 🆕 make sure to use the **development** branch
- for pull requests related to bug 🐛fixes of the existing stable release, make sure your pull request is going to the **release** branch.

Reply:
does that differentiation make sense? 

Do we have a separate branch for bug fixes of an existing release? e.g. other git conventions do something like:
- hot fix (for bug fixes that should go into both development and stable)
- development (or master)
- release (or stable)

Reply:
I think your explanation is good! The simpler we can keep it the better? For "hot fixes" that should really go to both `release` and `development` (which is probably pretty much everything?), maybe it's too much to expect new contributors to figure that out? I think it might be best to just have contributors pull request release bug fixes to `release` and we manually/internally keep the branches synced as necessary?

Reply:
@shiffman - sure! let's give this a try and adjust as we feel fit. Thanks for the feedback! I will commit my latest and merge in :)